lib/aftership/v4/configuration.rb

```diff
@@ -1,19 +1,24 @@
 module AfterShip
   module V4
     class Configuration
-      attr_accessor :api_endpoint, :api_key
-      attr_writer :timeout
+      attr_writer :connect_timeout, :send_timeout, :receive_timeout
 
       def initialize
         @api_endpoint = 'https://api.aftership.com'
-        @timeout = 3
         @headers = { 'Content-Type' => 'application/json' }
+        @connect_timeout = @send_timeout = @receive_timeout = 500 # milliseconds
       end
 
-      def timeout
-        return 0 unless @timeout
+      def connect_timeout
+        @connect_timeout.to_f / 1000 # to seconds
```

Comment:
we could create a private function which will be used for `{connect,send,receive}_timeout`

Something like:
```ruby
def to_seconds(value)
  value.to_f / 1000
end
```

WDYT?